Move travisci to Github Actions

aviquez96 opened the pull request:
- Create GHA script to replace .travis.yml file
- Adjust coverage report path in upstream.sh file
- Adjusted readme with new GHA badge
- Added push and pr behavior only on master

drazisil-codecov reviewed (changes requested):
https://github.com/codecov/typescript-standard/pull/17/checks?check_run_id=2671435352#step:4:76

Can you determine what credentials this needs?

Also, it should not succeed if this fails.

aviquez96 commented:
Hey @drazisil-codecov, based on the travis env vars, https://travis-ci.com/github/codecov/typescript-standard/settings, it should be API_KEY, CORRECT_COVERAGE and GH_TOKEN.

I'll look into to the false positive fail.

drazisil-codecov reviewed (changes requested):
Update upstream.sh to set the git identity and see why https://github.com/codecov/typescript-standard/pull/17/checks?check_run_id=2677040694#step:4:56 doesn't appear to run request.py to check the coverage and we should be good.

aviquez96 commented:
@drazisil-codecov, updated git identity and fixed coverage showing up https://github.com/codecov/typescript-standard/runs/2678672465?check_suite_focus=true#step:4:69. This value was hardcoded for the moment https://github.com/codecov/typescript-standard/pull/17/files#diff-54729f0e8b58320b1a35a5ca63adc4941bc1968b8b9881dee16fc8fcdf5e78c3R19, could you confirm it is added as a secret? I don't have access to view/edit them :(
